misc/scripts/refcnt_test.sh

```diff
@@ -79,10 +79,12 @@ function check_recovery_record {
 }
 
 function test_crash_recovery {
-    # Turning this off for PR #1047.
-    # TBD: bring it back when bringing discovery back (see PR #1050)
-    echo "Skippint crash recovery check (see #1050)... "
-    return
+    alive_containers = `docker ps | grep busybox | wc -l`
+    if [ $alive_containers -ne $count] ; then
+        echo "FAILED CRASH RECOVERY TEST. Not all test containers are running"
+        exit 1
+    fi
+    echo "$count containers are running with $vname attached"
```

Comment:
The way this test script works the timeouts for each of the containers has been kept so that they are still around by the time the plugin restarts and verifies the volume ref count. Does the alive_containers include the plugin also? Do we need this check at all?